common.sh

```diff
@@ -438,3 +438,9 @@ fi
 
 # Defaults the variable to enable testing a custom machine-api-operator image
 export TEST_CUSTOM_MAO=${TEST_CUSTOM_MAO:-false}
+
+# Asisted deployment specific params
```

Comment:
it would be better to put these in assisted_deployment.sh I think?

Reply:
will move